app/views/auctions/multi_bid/_win_header.html.erb

```diff
@@ -1,12 +1,12 @@
 <% if auction.over? %>
   <% if auction.bids? %>
     <% if current_user %>
-      <% if auction.user_is_winning_bidder?(current_user) %>
+      <% if auction.user_is_winning_bidder? %>
```

Comment:
This is an incredible amount of logic for a view. This was clearly here before, but if this massive change isn't improving situations like this, it is likely the wrong abstraction.